--- a/configure
+++ b/configure
@@ -16275,8 +16275,10 @@
 Usually this means the macro was only invoked conditionally." "$LINENO" 5
 fi
 if test -z "${HAVE_VALGRIND_TRUE}" && test -z "${HAVE_VALGRIND_FALSE}"; then
-  as_fn_error $? "conditional \"HAVE_VALGRIND\" was never defined.
-Usually this means the macro was only invoked conditionally." "$LINENO" 5
+  HAVE_VALGRIND_TRUE='#'
+  HAVE_VALGRIND_FALSE=
+
+
 fi
 if test -z "${HAVE_INTROSPECTION_TRUE}" && test -z "${HAVE_INTROSPECTION_FALSE}"; then
   as_fn_error $? "conditional \"HAVE_INTROSPECTION\" was never defined.
